src/commands/console/index.js

```diff
@@ -10,12 +10,184 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-const { Command } = require('@oclif/command')
+const aioConsoleLogger = require('@adobe/aio-lib-core-logging')('@adobe/aio-cli-plugin-console', { provider: 'debug' })
+const config = require('@adobe/aio-cli-config')
+const { Command, flags } = require('@oclif/command')
+const { getToken, context } = require('@adobe/aio-lib-ims')
+const sdk = require('@adobe/aio-lib-console')
+const { CLI } = require('@adobe/aio-lib-ims/src/context')
+const Help = require('@oclif/plugin-help').default
+const yaml = require('js-yaml')
 
-class IndexCommand extends Command {
+const { CONSOLE_API_KEY = 'TransporterJaeger1' } = process.env
```

Comment:
should be `aio-cli-console-auth`(prod) and `aio-cli-console-auth-stage` (stage) and possibly change according to the environment (some var)